File: src/main/java/org/n52/spare/kicker/repositories/MatchEventRepository.java
package org.n52.spare.kicker.repositories;

import org.n52.spare.kicker.model.MatchEvent;
import org.springframework.data.repository.PagingAndSortingRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface MatchEventRepository extends PagingAndSortingRepository<MatchEvent, Long> {

}

File: src/main/kotlin/org/n52/spare/kicker/repositories/MatchEventRepository.kt
package org.n52.spare.kicker.repositories

import org.n52.spare.kicker.model.MatchEvent
import org.springframework.data.repository.PagingAndSortingRepository
import org.springframework.stereotype.Repository

@Repository
interface MatchEventRepository : PagingAndSortingRepository<MatchEvent, Long>
